pallets/nfts/src/tests.rs

```diff
@@ -1954,49 +1954,110 @@ fn approval_lifecycle_works() {
 }
 
 #[test]
-fn check_approval_without_deadline_works() {
+fn check_approval_works_without_deadline_works() {
 	new_test_ext().execute_with(|| {
 		let collection_id = 0;
-		let item_id = 42;
 		let collection_owner = account(1);
+		let item_id = 42;
```

Comment:
Note it here because I think my variables did not order alphabetically. Will double check to reorder them!